### ISSUE TITLE
UUID issues on Scanning or connecting a certain BLE from several iOS devices

### ISSUE THREAD

**Issue by Younghun-Jung** [don/cordova-plugin-ble-central#661]
Apple iOS devices provide UUID when certain BLE is scanned or connected.
I got a issue about this. Assume that below scenario.

There are devices like below.
- 'A' iOS device
- 'B' iOS device
- 'foo' BLE

'A' iOS device scanned (or connected with) a 'foo' BLE and we can get UUID of 'foo'.

And then(after BLE 'foo' is disconnected with 'A' iOS device),

'B' iOS device scanned (or connected with) a 'foo' BLE and we can also get UUID of 'foo'.

Two UUIDs are different although two iOS devices have connected same BLE 'foo'.

I need to same ID value from a certain BLE device scanned or connected by several iOS devices.
I want to manage BLE list on my server and need to identify each BLE.

In above scenario, I want to get result like this...
if an user registered BLE 'foo' on server using UUID from scanning or connecting, other user cannot register BLE 'foo' on server because BLE 'foo' information is already saved on DB of server. In order to get this scenario, I need same UUID of certain BLE scanned or connected by several iOS devices.

If I cannot use this UUID, How can I implement this scenario? Is there any value for this?

**Comment by droghio** [comment]
The Peripheral UUID used in place of a MAC address is specific to the iOS device. There is no way specific value across devices, and this UUID can change even on the same device. If I recall this was enacted as some sort of privacy protection for iOS.

TLDR: If you want to identify a single BLE periperhal across two+ devices you'll need to embed a serial number or the like in the advertisement. This is not a limitation with the plugin but one imposed by iOS.
